src/iDynFor/iDynTreePinocchioConversions.h

```diff
@@ -18,86 +18,100 @@ namespace iDynFor
 namespace details
 {
 
-// All this part is taken from https://github.com/stack-of-tasks/pinocchio/blob/v2.6.8/src/parsers/urdf/model.hxx
-// Unfortunatly, it is not public, so it can't be reused
-template<typename _Scalar, int Options>
-class iDynTreeModelVisitorBaseTpl {
-  public:
-    enum JointType {
-      REVOLUTE, CONTINUOUS, PRISMATIC, FLOATING, PLANAR
+// All this part is taken from
+// https://github.com/stack-of-tasks/pinocchio/blob/v2.6.8/src/parsers/urdf/model.hxx Unfortunatly,
+// it is not public, so it can't be reused
+template <typename _Scalar, int Options> class iDynTreeModelVisitorBaseTpl
+{
+public:
+    enum JointType
+    {
+        REVOLUTE,
+        CONTINUOUS,
+        PRISMATIC,
+        FLOATING,
+        PLANAR
     };
     typedef _Scalar Scalar;
-    typedef pinocchio::SE3Tpl<Scalar,Options> SE3;
-    typedef pinocchio::InertiaTpl<Scalar,Options> Inertia;
+    typedef pinocchio::SE3Tpl<Scalar, Options> SE3;
+    typedef pinocchio::InertiaTpl<Scalar, Options> Inertia;
     typedef Eigen::Matrix<Scalar, 3, 1> Vector3;
     typedef Eigen::Matrix<Scalar, Eigen::Dynamic, 1> Vector;
     typedef Eigen::Ref<Vector> VectorRef;
     typedef Eigen::Ref<const Vector> VectorConstRef;
-    virtual void setName (const std::string& name) = 0;
-    virtual void addRootJoint (const Inertia& Y, const std::string & body_name) = 0;
-    virtual void addFixedJointAndBody(
-        const pinocchio::FrameIndex & parentFrameId,
-        const pinocchio::SE3 & joint_placement,
-        const std::string & joint_name,
-        const pinocchio::Inertia& Y,
-        const std::string & body_name) = 0;
-    iDynTreeModelVisitorBaseTpl () : log (NULL) {}
-    template <typename T>
-    iDynTreeModelVisitorBaseTpl& operator<< (const T& t)
+    virtual void setName(const std::string& name) = 0;
+    virtual void addRootJoint(const Inertia& Y, const std::string& body_name) = 0;
+    virtual void addFixedJointAndBody(const pinocchio::FrameIndex& parentFrameId,
+                                      const pinocchio::SE3& joint_placement,
+                                      const std::string& joint_name,
+                                      const pinocchio::Inertia& Y,
+                                      const std::string& body_name)
+        = 0;
```

Comment:
As a personal taste, the ``= 0`` in the new line seems a bit weird, but I can live with it.

Reply:
I think the `[.clang-format](https://github.com/ami-iit/idynfor/blob/master/.clang-format)` in the repo comes from the blf repo, if there is any adjustement I think we can do it, otherwise I am not sure it is worth to manually address these cases